lua/nvim-tree.lua

```diff
@@ -470,6 +470,8 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
     remove_file = {
       close_window = true,
     },
+    delete_buffer = {},
+    wipe_buffer = {},
```

Comment:
Are these options used?

Reply:
No, I was unsure if it was a convention to have these even if there are no underlying options, so I added these in even though they are empty.

I can remove them 👍🏼